src/views/tutorial-view/utils/mdx-components.tsx

```diff
@@ -16,33 +13,16 @@ import {
   MdxOrderedList,
   MdxUnorderedList,
   MdxListItem,
-} from 'components/dev-dot-content/mdx-components/mdx-lists'
-import Tabs, { Tab } from 'components/tabs'
+  MdxTabs,
+  MdxTab,
+} from 'components/dev-dot-content/mdx-components'
 import Text from 'components/text'
 import VideoEmbed from 'components/video-embed'
 
-// Local imports
-import s from './mdx-components.module.css'
```

Comment:
💭 Looks like this `src/views/tutorial-view/utils/mdx-components.module.css` can also be removed.

Reply:
Ooh great catch, ty! Updated to remove ✅ 🔪